Makefile

```diff
@@ -82,13 +82,17 @@ packer-linux-amd64.output: $(PACKER_LINUX_FILES)
 		-e AWS_ACCESS_KEY_ID \
 		-e AWS_SECRET_ACCESS_KEY \
 		-e AWS_SESSION_TOKEN \
+		-e SOURCE_AMI_NAME_FILTER \
+		-e SOURCE_AMI_OWNER \
 		-e PACKER_LOG \
 		-v ${HOME}/.aws:/root/.aws \
 		-v "$(PWD):/src" \
 		--rm \
 		-w /src/packer/linux \
 		hashicorp/packer:$(PACKER_VERSION) build -timestamp-ui -var 'region=$(AWS_REGION)' \
 			-var 'arch=x86_64' -var 'goarch=amd64' -var 'instance_type=$(AMD64_INSTANCE_TYPE)' \
+			-var 'source_ami_name_filter=$(SOURCE_AMI_NAME_FILTER)' \
```

Comment:
Does this override the default value from the `variables` section with an empty string if the environment variable isn’t set?